README.md

```diff
@@ -1,98 +1,89 @@
-# Underdog Devs Devops Class I
+# **Underdog Devs Devops Class I**
 
-We want a local vm that is Ubuntu that is running an nginx webserver
-from
-local
-windows
-osx
-ubuntu
+## Goal- create local Ubuntu VM running an NginX webserver from:
+* Windows
+* Osx
+* Ubuntu
 
-# usage
+&nbsp;
 
-install git
+# Usage
 
-clone the repo
+- install git
 
-install a hypervisor
+- clone the repo
 
-write a file or command the provisions a ubuntu vm
+- install a hypervisor
 
-connect to the vm
+- write a file or command the provisions an ubuntu vm
 
-install the webserver
+- connect to the vm
 
-test the webserver
+- install the webserver
+
+- test the webserver
+
+&nbsp;
 
 ## **Windows**
```

Comment:
We should change the README.md to just showing how to run the appropriate script for your system:

Windows
`webserver.ps1`

Mac:
`zsh webserver.zsh`

...

Reply:
Changes made to README here https://github.com/curl-bash/sturdy-octo-doodle/tree/jdoze/fullWIP